src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -258,13 +258,13 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
 
   def checkUserInRawls(
     pipeline: WithTransformerConcatenation[HttpRequest, Future[HttpResponse]],
-    requestContext: RequestContext): Future[PerRequestMessage] = {
+    requestContext: RequestContext, userId: String): Future[PerRequestMessage] = {
```

Comment:
The spray pipeline is created at line 64, and includes the Authorization: header from the original request. That pipeline is passed into checkUserInRawls (line 260). So, when orch calls rawls, it passes the Authorization: header to rawls. Then, the rawls proxy can identify the user, just from that header.